src/components/NewsArticles.jsx

```diff
@@ -0,0 +1,26 @@
+import { Card } from "antd";
+
+function NewsArticles({article}) {
+
+    return (
+        <Card
+            title={article.title}
+            style={{
+                height: 500,
+                width: 550,
+                overflow: 'hidden'
+            }}
+        >
+            <p>Author: {article.author}</p>
+            <p>Topic: {article.topic}</p>
+            <div style={{width: '100%', height: '200px', overflow: 'hidden'}}>
+                {article.article_img_url ? <img style={{width: '100%', height: 'auto', overflow: 'hidden'}} src={article.article_img_url}/>: null}
+            </div>
+            <button>Expand</button>
```

Comment:
The button does look a _bit_ out of place. As an alternative you could turn part of, or all of the card into a link as a way to navigate to an individual article instead, or if you're set on the button idea, play around with the styling